### ISSUE TITLE
Issues with Angular 8.2.2

### ISSUE THREAD

**Issue by cgeffre** [LaunchCodeEducation/intro-to-professional-web-dev#392]
As the version of Angular used in Intro to Professional Web Development has fallen further out of date, many students (if not a majority or all) in the recent LC101 Philly cohort have encountered new compatibility issues when installing Angular version 8.2.2 (as required by the textbook: https://education.launchcode.org/intro-to-professional-web-dev/chapters/angular-lsn1/file-structure.html). Version 8.2.2 no longer appears to be compatible with the most recent version of npm, causing Angular to be unable to run. The most common method of solving this has been for students to downgrade npm with "sudo npm install -g npm@6.14.15" (or removing sudo, in some cases). Rather than needing students to downgrade npm, it would probably be best if a more recent edition of Angular was implemented as the standard for the textbook going forward.

**Comment by a-zabala** [comment]
I think this is the same issue as we are experiencing in Women + currently.  This is the info I got from a learner and I do not know how to fix it.  But we definitly need to include instructions on what to do, as we will have many learners trying to figure this out, and our Angular portion has been condensed to two weeks.  This is what the learner said:  The textbook wants the students to install Angular CLI 8.2.2 (which is 3 years old) in order for us to be able to follow the instructions as they are laid out in the textbook. As you know, I (and several other students) had issues with it not wanting to install due to conflicts with the version of Jasmine we already have installed on our computers.
We were able to get Angular to install if we did the most recent version (14, which came out a month ago). But now we're no longer able to follow along with the out-of-date instructions in the textbook.
